spec/unit/recipes/default_spec.rb

```diff
@@ -46,7 +46,7 @@
 
   describe 'test::default on Fedora' do
     let(:chef_run) do
-      ChefSpec::SoloRunner.new(platform: 'fedora', version: '31').converge('test::default')
+      ChefSpec::SoloRunner.new(platform: 'fedora', version: '32').converge('test::default')
```

Comment:
Long term just take the version out here to avoid having to fix this every 6 months